lib/src/se/llbit/math/QuickMath.java

```diff
@@ -159,4 +159,19 @@ public static float abs(float x) {
   public static double abs(double x) {
     return (x < 0.0) ? -x : x;
   }
+
+  /**
+   * Greatest Common Divisor of both inputs
+   */
+  public static int gcd(int number1, int number2) {
+    if (number1 == 0 || number2 == 0) {
+      return number1 + number2;
+    } else {
+      int absNumber1 = Math.abs(number1);
+      int absNumber2 = Math.abs(number2);
+      int biggerValue = Math.max(absNumber1, absNumber2);
+      int smallerValue = Math.min(absNumber1, absNumber2);
+      return gcd(biggerValue % smallerValue, smallerValue);
+    }
```

Comment:
(nit) Would a non-recursive algorithm, i.e. a loop that updates number1 and number 2, be faster?